### DISCUSSION TITLE
Where can I find older versions of Artisan?

### DISCUSSION THREAD

**Discussion by coraxxos** [artisan-roaster-scope/artisan#689] [Q&A]
Hi.

I am looking for a version of Artisan compatible with  macOS 10.13, where can I find that?

All the best

Coraxos

**Comment by MAKOMO** [comment]
Click on "All Releases" on artisan-scope.org

**Comment by MAKOMO** [comment]
Note that it is not recommended to run older releases in general. We can only support the current release for resource reasons.
